### DISCUSSION TITLE
digikey parts list

### DISCUSSION THREAD

**Discussion by RaymondDay61** [frntc/SIDKick-pico#29] [Ideas]
If you go here can make a digikey parts list and then know will get the right parts. https://www.digikey.com/en/mylists/

I was doing it and it's hard to find some. I don't know if it's the right size resistors and things like that so it would be good for some one to make a list there so can just click it on to order all the parts.

Got the boards from PCB way from the link on here and I thought I order them with the parts all ready on them but I got them blank they look real good but now I have to get the parts and solder them on it.

-Raymond Day

**Comment by frntc** [comment]
Have you seen the BOM on the PCBWay page? It has all the data. 

If you're looking for alternative parts (the part numbers in the BOM are from Mouser): 
- SN74CBTD3861DBQR is a unique part
- resistors: any thick film resistor with the right resistance and 0805 footprint does the job
- capacitors: any 100nF in 0805
- diode: this is the most "tricky" thing. This one from Digikey should work: https://www.digikey.de/en/products/detail/taiwan-semiconductor-corporation/TS4148-RBG/15926409 (as 1n4148 in 0805 footprint are a bit rare, you can also try a SOD-323 footprint with handsoldering)

**Reply by RaymondDay61** [reply]
Are you saying there is a link to Mouser for all the parts some were?

**Comment by RaymondDay61** [comment]
That is only the diode link.

My cart looks like this but I think don't have the right parts yet.

I copied the page and put it on my google drive here.

https://photos.app.goo.gl/wFk5PyvzKnDFzdmW8

It be nice if could just click on a link to have all the parts listed there to buy. Or any link were they sell all the parts for this board?

-Raymond Day

**Reply by frntc** [reply]
The resistors and diodes in your cart are not SMD parts. The BOM on the PCBWay page has Mouser part numbers, you can use those to retrieve the actual values. (The diode is no longer available from Mouser, you can take one with the values in the BOM or the one linked to above)

**Reply by RaymondDay61** [reply]
I did not get the right resistor and I guess not the diode ether. Did order the right resistor and waiting for it. This the the first time I ever soldered surface mountain Parts they are small I got a photo of it on my Google photos here https://photos.app.goo.gl/2yoDnaw7ztWVXFpP8

Going to look were I can order the diode it's not at Mouser.

One thing I want to do this is stereo do I need the DAC board to do this?

Don't see any place to solder left and right wires on it.

Thank you.

Just order some pin headers I guess they are 2.54 mm that is one thing the read me don't say.

Looked on line and order the diodes from here:

Adafruit
Products
------------------------------------------------------
1 x 1N4148 SMT SOD-123 Diodes - 100 Pack[ID:5099] = $4.95
------------------------------------------------------
Sub-Total: $4.95
USPS (1 pkg x 0.05lbs) (USPS - Ground Advantage): $4.61
Sales Tax: $0.30
Total: $9.86

Should be about all set after I get the parts now.

-Raymond Day

**Comment by RaymondDay61** [comment]
I got the 5 other boards they all ready had the parts on them. So I solder the pins to it. Install it on the old C=64 and I got this photo of it in the Commodore on the google photos.

https://photos.app.goo.gl/UMSXq1jczH7ybaQo8

Were does a wire go for the 2nd SID for stereo I want to solder it on Pin 7 of the video port.

Not sure if this old C=64 works When I plug it in the power LED did not go on. So I unsolder the 2 address pins from the board and then the power LED goes on but no video the screen turns black like it's trying. Maybe I messed it up must of solder the 2 address lines on the wrong pins on the ROM chip. I got another old Commodore 64 and will get it out to try on that but don't have time today will try tomorrow.

Looks like the PLA the 906114-01 U17 went bad in this. Order a replacement from 8-Bit Classics.

Thank you.

-Raymond Day

**Comment by DariBer** [comment]
@frntc Is there a BOM like the one for 0.2 PCB at PCBWAY, but for the 0.1 PCB?
I orderd  PCB from PCBWAY, then it was the 0.1 that was able, but unfortunatly I didn't have time to source the components, so would be nice to have the old BOM with mgf partnumber.

**Comment by frntc** [comment]
Hey, the BOM is identical for the non-DAC-builtin-version:

| Manufacturer      | *Mfg Part #       | Description / Value                                        | *Package/Footprint     |
|-------------------|-------------------|------------------------------------------------------------|------------------------|
| YAGEO             | CC0805KRX7R9BB104 | 50V 100nF X7R ±10% 0805 Multilayer Ceramic Capacitors MLCC | 0805                   |
| Texas Instruments | SN74CBTD3861DBQR  | SN74CBTD3861DBQR                                           | SSOP-24-150mil-0.635mm |
| UNI-ROYAL         | 0805W8F1801T5E    | 125mW Thick Film Resistors ±1% 1.8kOhm                     | 0805                   |
| UNI-ROYAL         | 0805W8F4700T5E    | 125mW Thick Film Resistors ±1% 470Ohm                      | 0805                   |
| UNI-ROYAL         | 0805W8F1002T5E    | 125mW Thick Film Resistors ±1% 10kOhm                      | 0805                   |
| LIZ Elec          | CD4148WSP         | 75V 400mW 1.25V@100mA 4ns 150mA Switching Diode            | 0805                   |

Most of the values do not need to be matched exactly, e.g. the 1.8kR can also be 2kR, also the 10kR is not critical, other voltages for the caps are OK, and also other 4148 diodes should work.

**Reply by DariBer** [reply]
Thanx for the list. I found everything except the diod, can you recommend any that is common and can be found on Mouser or Digikey?

**Reply by DariBer** [reply]
@frntc will this work, TS4148 RBG?

[https://mou.sr/3zZRtb3](url)

**Reply by frntc** [reply]
It should work. I changed the diode's footprint from 0805 to SOD323 because the 0805 are difficult to find (as you noticed). If you solder yourself, you can also fit SOD323, e.g. A1N4148WS-HF.
